### PR TITLE
[FIX] records: skip inactive, invalid views in adapt_domains()

### PR DESCRIPTION
**Pull request by cawo-odoo**
Do not fail when the XML parser fails to parse an inactive view. Inactive views are probably abandoned and do not cause immediate errors in the upgraded db, so just skip them when they are invalid.

Also, log the view_id before (re)raising a parser exception. This should save some time when manual fixing of invalid (active) views is needed.

upg-1171125

### CODE REVIEW

File: src/util/domains.py [hunk 1]
```diff
@@ -354,6 +354,7 @@ def adapt_domains(cr, model, old, new, adapter=None, skip_inherit=(), force_adap
                 # this view is already wrong, we don't change it
                 _logger.warning("Skipping domain adpatation for invalid view (id=%s):\n%s", view_id, e.msg)
                 continue
+            _logger.exception("Failing domain adpatation of invalid view (id=%s)", view_id)
```
Comment (aj-fuentes) [inline src/util/domains.py:357]: There was a typo here and the warning above: `adpatation`
Sill I'm not sure if we want to log the whole error... the view is already broken, we are not supposed to fix it. We fix views that are correct but do not work due to something done during the upgrade.

File: src/util/domains.py [hunk 2]
```diff
@@ -350,10 +350,16 @@ def adapt_domains(cr, model, old, new, adapter=None, skip_inherit=(), force_adap
...
                 # this view is already wrong, we don't change it
-                _logger.warning("Skipping domain adpatation for invalid view (id=%s):\n%s", view_id, e.msg)
+                _logger.warning(
+                    "Skipping domain adaptation for %sinvalid view (id=%s):\n%s",
+                    "" if view_active else "inactive, ",
+                    view_id,
+                    e.msg,
+                )
                 continue
+            _logger.error("Failing domain adaptation of invalid view (id=%s)", view_id)  # noqa: TRY400
```
Comment (aj-fuentes) [inline src/util/domains.py:362]:
```suggestion
            _logger.error("Cannot adapt domain of invalid view (id=%s)", view_id)  # noqa: TRY400
```

**Comment by KangOl** [comment]
upgradeci retry with always only crm

**Comment by aj-fuentes** [comment]
@robodoo r+

**Pull request closed without merging**